### DISCUSSION TITLE
Can we support China's self-developed CPU architecture c-sky

### DISCUSSION THREAD

**Discussion by maweicoltsmart** [zephyrproject-rtos/zephyr#68085] [Q&A] [answered]
Linux already supports China's self-developed CPU architecture C-SKY. Can it be ported to Zephyr？

**Comment by henrikbrixandersen** [comment]
I am sure, it can - however, someone needs to put in the work and maintain it going forward. Are you interested in porting it yourself and maintaining it for the years to come?

**Comment by maweicoltsmart** [comment]
> I am sure, it can - however, someone needs to put in the work and maintain it going forward. Are you interested in porting it yourself and maintaining it for the years to come?

haha, Sorry, this job is beyond my capabilities. Does the organization have any plans to support the c-sky CPU architecture

**Comment by henrikbrixandersen** [comment, marked as answer]
> haha, Sorry, this job is beyond my capabilities. Does the organization have any plans to support the c-sky CPU architecture

The Zephyr Project is an open source project, and as such relies on someone to step up and do the work and volunteer to maintain it. I am not aware of anybody planning to add support for C-Sky, sorry.
